pkg/bpf/bpffs_windows.go

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package bpf
+
+import (
+	"errors"
+)
+
+var (
+	notSupportedWinErr = errors.New("not supported on windows")
+)
+
+// mountFS mounts the BPFFS filesystem into the desired mapRoot directory.
+func mountFS(root, kind string) error {
```

Comment:
I think this function can be removed, it's not called from any windows place or common code

Reply:
I have removed this and verified compile